main.go

```diff
@@ -223,6 +223,8 @@ func runCLIApp(c *cli.Context) (err error) {
 	config.OverrideWithLoggingFlags(mountConfig, flags.LogFile, flags.LogFormat,
 		flags.DebugFuse, flags.DebugGCS, flags.DebugMutex)
 
+	logger.SetLogFormat(mountConfig.LogConfig.Format)
```

Comment:
Yeah, looks a bit like the chicken and egg problem. 

For now, please add a comment here saying "Ideally this call to SetLogFormat (which internally creates a new defaultLogger) should be set as an else to the 'if ForeGround' check below, but currently that means the logs generated by resolveConfigFilePaths below don't honour the user-provided log-format", so that if this can be fixed later, it should be.